muclock/src/server.ts

```diff
@@ -39,17 +37,16 @@ class MuClockClientPingHandler {
 
         // do ping operation
         this.pingCount += 1;
-        this.lastPingStart = startClock;
-        this.lastPingUUID = Math.floor(Math.random() * 1e10);
-        this.protocolClient.message.ping(this.lastPingUUID);
-        (function(startPingClock, env) {
+        this._lastPingStart = startClock;
+        this._lastPingUUID = Math.floor(Math.random() * 1e10);
+        this.protocolClient.message.ping(this._lastPingUUID);
+        (function(startPingClock, pingHandler) {
             setTimeout(
```

Comment:
This is not efficient.  We will end up creating many timeout events and closures each time we process a ping.  Instead we can do the check for timeouts when poll is called.